### PR TITLE
Technotes for the nomisr package

### PR DESCRIPTION
**Pull request by evanodell**

### CODE REVIEW

File: content/technotes/2018-04-27-nomisr-access-uk-census-data.Rmd [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: "Nomisr - Access 'Nomis' UK Labour Market Data"
```
Comment (stefaniebutland) [inline content/technotes/2018-04-27-nomisr-access-uk-census-data.Rmd]: Please add slug and topicid to YAML e.g.
`slug: "nomisr"`
`topicid: `

I will add the topicid at the time of publication.

Comment (stefaniebutland) [reply content/technotes/2018-04-27-nomisr-access-uk-census-data.Rmd]: Please knit the .Rmd to .md for rendering on our site.

File: content/technotes/2018-04-27-nomisr-access-uk-census-data.Rmd [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+
+
+## To do
+
+The `nomisr` package covers most of the features available on Nomis, with the exception of a grouping function that can easily be replicated in R, and that will likely be incorporated into a future release. Long term development is focused on making the package more user-friendly for people with less familiarity with Nomis data and the API structure.
+
+### Package review
+
+Thank you to reviewers [Paul Egeler](https://github.com/pegeler) and [Christophe Dervieux](https://github.com/cderv) for their incredibly helpful comments and suggestions, and for [Karthik Ram](https://github.com/karthik)'s editorial supervision. `nomisr` is a much better package than would otherwise have been the case as a result of their efforts.
```
Comment (stefaniebutland) [inline content/technotes/2018-04-27-nomisr-access-uk-census-data.Rmd]: consider adding link to the open review thread: https://github.com/ropensci/onboarding/issues/190

File: content/technotes/2018-04-27-nomisr-access-uk-census-data.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+## Using `nosmisr`
+
+The [vignette](https://cran.rstudio.com/web/packages/nomisr/vignettes/introduction.html) has more details on identifying available datasets and the parameters they accept, but the example below shows the latest data on Job Seekers Allowance claimants in each parliamentary constitutency. 
+
+
+```{r}
+library(nomisr)
+
+jsa_constits <- nomis_get_data(id = "NM_1_1", time = "latest",
```
Comment (sckott) [inline content/technotes/2018-04-27-nomisr-access-uk-census-data.md]: do you want this code block to be rendered in the post? or do you just want to show the code block here as is?

**Comment by stefaniebutland** [comment]
@evanodell In order to elevate this post beyond what's in the README, we have some suggestions. Best to skip installation instructions, and pointers to README. All are in the repo. This should tell a very specific story, like a technical challenge or cool issue this package solves. How do **you** use `nomisr`? Do you use it in combination with other tools?

Let me know if you're interested in writing it that way, or would rather not publish.

File: content/technotes/2018-04-27-nomisr-access-uk-census-data.md (hunk 3 above)
Comment (evanodell) [reply content/technotes/2018-04-27-nomisr-access-uk-census-data.md]: Ideally I'd like it rendered, but knitting from Rmd into md hasn't worked for me.

**Comment by karthik** [comment]
@evanodell Can you update your setup chunk to this:

```r
knitr::opts_chunk$set(
  comment = "#>",
  collapse = TRUE,
  warning = FALSE,
  message = FALSE,
  upload.fun = knitr::imgur_upload, base.url = NULL
)
```

Your map should show correctly after that.

**Comment by stefaniebutland** [comment]
@evanodell Please let me know when you have made the suggested edits and are happy with the narrative so I can do a final review and publish quickly after that.

**Comment by evanodell** [comment]
@stefaniebutland I've pushed an updated version now.

**Comment by stefaniebutland** [comment]
Map still not rendering in preview @evanodell. I think you need to update the date in YAML (currently: `date: 2018-04-30`). After you update, have a look here and confirm the map shows up: https://deploy-preview-175--ropensci.netlify.com/technotes/2018/05/08/nomisr/

Almost there!

**Comment by evanodell** [comment]
@stefaniebutland thank you for pointing that out. And the map is now rendering!

**Comment by stefaniebutland** [comment]
published and tweeted! Thanks for your patience with the back and forth. I hope the Tech Note gets some extra eyes on your work

**Pull request merged**